### PR TITLE
Kenya wards

### PR DESCRIPTION
**Pull request by DanielMwendwa**
[Source](https://drive.google.com/drive/u/0/folders/1dPWDsJ3Sfm1180t4dy-fy8X3bPV_eMj4)

### CODE REVIEW

**Comment by DanielMwendwa** [comment]
> Thanks @DanielMwendwa. Please fix the failing tests and comments and request re-review.

This PR is part of [map-ward-to-const](https://github.com/AfricasVoices/CoreDataModules/tree/map-ward-to-const) which has passed all the tests. Added changes in separate PR's to ease review. I'll merge branch [map-ward-to-const](https://github.com/AfricasVoices/CoreDataModules/tree/map-ward-to-const) after approval.

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 1]
```diff
@@ -343,92 +343,1501 @@ class KenyaCodes(object):
...
-        MUVUTINI_KIIMA_KIMWE: MACHAKOS_TOWN,
-        KOLA: MACHAKOS_TOWN,
-        MBIUNI: MWALA,
-        MAKUTANO: MWALA,
-        MASII: MWALA,
-        MUTHETHENI: MWALA,
-        WAMUNYU: MWALA,
-        KIBAUNI: MWALA
-    }
+    # WARD_TO_CONSTITUENCY_MAP = {
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply core_data_modules/cleaners/codes/kenya_codes.py:1799]: Yes, changes in subsequent PRs.

**Comment by DanielMwendwa** [comment]
> > > Thanks @DanielMwendwa. Please fix the failing tests and comments and request re-review.
> > 
> > 
> > This PR is part of [map-ward-to-const](https://github.com/AfricasVoices/CoreDataModules/tree/map-ward-to-const) which has passed all the tests. Added changes in separate PR's to ease review. I'll merge branch [map-ward-to-const](https://github.com/AfricasVoices/CoreDataModules/tree/map-ward-to-const) after approval.
> 
> Changes should be incremental and not break main.

https://github.com/AfricasVoices/CoreDataModules/pull/245 contains all the changes. You can review that instead.

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 2]
```diff
@@ -343,92 +343,1501 @@ class KenyaCodes(object):
...
+    UPPER_SAVANNAH = "upper savannah"
+    URAN = "uran"
+    USONGA = "usonga"
+    UTALII = "utalii"
+    UTAWALA = "utawala"
+    UTHIRU = "uthiru"
+    UTHIRU_RUTHIMITU = "uthiru/ruthimitu"
+    VANGA = "vanga"
+    VISOI = "visoi"
+    VIWANDANI_MAKADARA = "viwandani-makadara"
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply core_data_modules/cleaners/codes/kenya_codes.py]: Updated, thanks.

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 3]
```diff
@@ -343,92 +343,1501 @@ class KenyaCodes(object):
     YATTA = "yatta"
 
     # Kenya Wards
```
[the remark this answers is not in the document]
Comment (DanielMwendwa) [reply core_data_modules/cleaners/codes/kenya_codes.py]: Updated, thanks.

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 4]
```diff
@@ -342,52 +342,1829 @@ class KenyaCodes(object):
     WUNDANYI = "wundanyi"
     YATTA = "yatta"
 
-    # Kenya Wards
-    # Machakos County Wards, src: https://kenyayote.com/iebc-machakos-county-registered-voters-constituency-wards/
+    # Kenya Wards src: https://docs.google.com/spreadsheets/d/1NQEJ7ikjrmqQAI8sJ-SXyoZhvOYftMtY/edit?usp=share_link&ouid=105008405800104279592&rtpof=true&sd=true
```
Comment (as2388) [inline core_data_modules/cleaners/codes/kenya_codes.py:345]: Can you add another note describing where this spreadsheet was retrieved from or how it was generated?

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 5]
```diff
@@ -304,7 +303,8 @@ class KenyaCodes(object):
     SOUTH_MUGIRANGO = "south mugirango"
     SOY = "soy"
     STAREHE = "starehe"
-    SUBA = "suba"
+    SUBA_NORTH_MBITA = "suba north/mbita"
```
Comment (as2388) [inline core_data_modules/cleaners/codes/kenya_codes.py:306]: Is this a renamed constituency or is it now officially known as "Suba North/Mbita"?

If it was renamed and is now only referred to as "Suba North" officially, then I suggest dropping the "Mbita".

File: core_data_modules/cleaners/codes/kenya_codes.py [hunk 6]
```diff
@@ -83,6 +83,7 @@ class KenyaCodes(object):
     CHEPALUNGU = "chepalungu"
     CHERANGANY = "cherangany"
     CHESUMEI = "chesumei"
+    CHUKA_IGAMBANG_OMBE = "chuka/igambang'ombe"
```
Comment (as2388) [inline core_data_modules/cleaners/codes/kenya_codes.py:86]: see my mbita comments

**Review by as2388 — APPROVED** [review]

File: core_data_modules/cleaners/codes/kenya_codes.py (hunk 5 above)
Comment (DanielMwendwa) [reply core_data_modules/cleaners/codes/kenya_codes.py]: It was renamed and is now officially referred to as "Suba North". I've dropped "Mbita".

File: core_data_modules/cleaners/codes/kenya_codes.py (hunk 6 above)
Comment (DanielMwendwa) [reply core_data_modules/cleaners/codes/kenya_codes.py:86]: Const. "Igambang'ombe" is now officially referred as "chuka/igambang'ombe". leaving as is.

File: core_data_modules/cleaners/codes/kenya_codes.py (hunk 4 above)
Comment (DanielMwendwa) [reply core_data_modules/cleaners/codes/kenya_codes.py]: Updated thanks.

**Pull request merged**